app/registries/foreman/access_permissions.rb

```diff
@@ -756,4 +756,8 @@
     map.permission :edit_settings, { :settings => [:update],
                                      :'api/v2/settings' => [:update] }
   end
+
+  permission_set.security_block :react do |map|
+    map.permission :view_react, {:react => [:index]}
```

Comment:
I'm confused, why is this needed?
Do we need special permission to see react pages or it should just be allowed to everyone?

Reply:
these changes are needed for a rake test to pass (permissions_test, seed_test)

Reply:
They shouldn't be. We don't add permissions just so tests pass when they don't make sense from a user standpoint. opened https://projects.theforeman.org/issues/27333 to remove this.